pkg/controller/registry/resolver/source_registry.go

```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/blang/semver/v4"
 	"github.com/operator-framework/operator-lifecycle-manager/pkg/controller/registry"
 	"github.com/operator-framework/operator-lifecycle-manager/pkg/controller/registry/resolver/cache"
+	"github.com/operator-framework/operator-registry/alpha/declcfg"
```

Comment:
Let's avoid this dependency. OLMv0 is unaware of FBC concepts/schemas, etc.

Like I said in another comment, I think we can just have `Package`, `Channel` and `Bundle`/`CSVName` fields for each deprecation layer.